app/assets/javascripts/discourse/app/lib/plugin-api.js

```diff
@@ -1993,35 +1992,6 @@ class PluginApi {
   registerModelTransformer(modelName, transformer) {
     registerModelTransformer(modelName, transformer);
   }
```

Comment:
This was removed in favour of using `register_hashtag_type_in_context` in the plugin API, since then we can just register once in the backend and use that throughout the whole app